apps/bfd-model/bfd-model-codegen/src/main/java/gov/cms/bfd/model/codegen/RifLayoutsProcessor.java

```diff
@@ -1892,6 +1910,20 @@ private static List<RifField> createDetailsForAdditionalDatabaseFields(
         addlDatabaseFields.add(lastUpdated);
         continue;
       }
+      if (additionalDatabaseField.contentEquals("BENE_ID_NUMERIC")) {
+        RifField lastUpdated =
```

Comment:
I see the same error in the other fields, but this var shouldnt be named lastUpdated since this is bene_id_numberic.